digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/MassTransitStopExcelDataImporter.scala

```diff
@@ -1,199 +1,199 @@
-    if (isBlank(value)) return
-
-    val propertyId = sql"""
-      select id from text_property_value
-      where property_id = (select p.id from property p where p.public_id = ${propertyPublicId})
-      and asset_id = ${assetId}
-    """.as[Long].firstOption
-
-    propertyId match {
-      case None => {
-        logger.info("  CREATING PROPERTY VALUE: '" + propertyPublicId + "' WITH VALUE: '" + value + "'")
-        sqlu"""
-          insert into text_property_value(id, property_id, asset_id, value_fi, created_by)
-          values (primary_key_seq.nextval, (select p.id from property p where p.public_id = ${propertyPublicId}), ${assetId}, ${value}, ${Updater})
-        """.execute
-      }
-      case _ => {
-        logger.info("  UPDATING PROPERTY VALUE: '" + propertyPublicId + "' WITH VALUE: '" + value + "'")
-        sqlu"""
-          update text_property_value set value_fi = ${value}, modified_by = ${Updater}, modified_date = SYSDATE
-          where id = ${propertyId}
-        """.execute
-      }
-    }
-  }
-
-  private def setSingleChoiceProperty(assetId: Long, propertyName: String, status: Option[Status]) {
-    status match {
-      case None =>  {
-        logger.info("  NOT UPDATING SINGLE CHOICE VALUE: " + propertyName)
-      }
-      case _ => {
-        logger.info("  SETTING SINGLE CHOICE VALUE: '" + propertyName + "' TO: " + status.get.dbValue)
-
-        val propertyId = sql"""select p.id from property p where public_id = ${propertyName}""".as[Long].first
-        val existingProperty = sql"""select property_id from single_choice_value where property_id = ${propertyId} and asset_id = ${assetId}""".as[Long].firstOption
-
-        existingProperty match {
-          case None => {
-            sqlu"""
-              insert into single_choice_value(property_id, asset_id, enumerated_value_id, created_by, created_date)
-              values (${propertyId}, ${assetId}, (select id from enumerated_value where value = ${status.get.dbValue} and property_id = ${propertyId}), ${Updater}, SYSDATE)
-            """.execute
-          }
-          case _ => {
-            sqlu"""
-              update single_choice_value set enumerated_value_id = (select id from enumerated_value where value = ${status.get.dbValue} and property_id = ${propertyId}), modified_by = ${Updater}, modified_date = SYSDATE
-              where property_id = $propertyId and asset_id = ${assetId}
-            """.execute
-          }
-        }
-      }
-    }
-  }
-}
-
-object MassTransitStopExcelDataImporter extends App {
-  val importer = new MassTransitStopExcelDataImporter(initDataSource)
-  importer.parseStopDataFromCsvFile("pysakkitiedot.csv")
-
-  private[this] def initDataSource: DataSource = {
-    Class.forName("oracle.jdbc.driver.OracleDriver")
-    val ds = new BoneCPDataSource()
-    ds.setJdbcUrl("")
-    ds.setUsername("")
-    ds.setPassword("")
-    ds
-  }
-}
-
+//package fi.liikennevirasto.digiroad2.util
```

Comment:
Can this all be deleted?

Reply:
I agree

Reply:
Verify first the reason of that, and then delete it if is not needed anymore